tutorials/3d/using_transforms.rst

```diff
@@ -46,7 +46,7 @@ Following is a visualization of rotation axes (in X,Y,Z order) in a gimbal (from
 
 You may be wondering how this affects you. Let's look at a practical example:
 
-Imagine you are working on a first-person controller (FPS game). Moving the mouse left and right controls your view angle parallel to the ground, while moving it up and down moves the player's view up and down.
+Imagine you are working on a first-person shooter (FPS game). Moving the mouse left and right controls your view angle parallel to the ground, while moving it up and down moves the player's view up and down.
```

Comment:
Actually "first-person controller" is correct, it's the type of character controller used for any first-person game (including FPS, but not limited to shooters). So it's more the "(FPS game)" that should be changed, maybe "(e.g. a FPS game)".

Reply:
@akien-mga : Taken care of.